.gitignore

```diff
@@ -63,6 +64,9 @@ instance/
 # Scrapy stuff:
 .scrapy
 
+# PyCharm
+\.idea/
```

Comment:
a file renamed from gitignore to gitignore
my gitignore does not work without \. instead.